site/profile/manifests/ccs/tomcat.pp

```diff
@@ -80,6 +85,19 @@
     }
   }
 
+  unless (empty($jars)) {
```

Comment:
The unless isn't needed here as `{}.each` is a noop.

Reply:
That's what I get for reusing the [existing style](https://github.com/lsst-it/lsst-control/blob/production/site/profile/manifests/ccs/tomcat.pp#L74)...

I have pushed a revised version.

Reply:
Ya. You can't trust that guy.